generators/app/templates/common/_package.json

```diff
@@ -4,17 +4,17 @@
   "description": "<%= description %>",
   "main": "index.js",
   "engines": {
-    "node": ">=6"
+    "node": ">=10"
   },
   "devDependencies": {
-    "babel-cli": "^6.22.2",
-    "babel-core": "^6.22.1",
-    "babel-preset-es2015": "^6.22.0",
-    "eslint": "^3.15.0"  
+    "@babel/cli": "^7.7.0",
```

Comment:
Here we should use https://github.com/verdaccio/monorepo/tree/master/tools/babel-preset#verdacciobabel-preset 

so all plugins should be based on this config at `.babelrc`

```
{
  "presets": [["@verdaccio"]]
}
```
Simple as that.

Reply:
Furthermore, this will help plugin developers to have a simple setup of dependencies, as `react-scripts` does, the same with `eslint` rules.

Reply:
should I change the preset to verdaccio then ?

Reply:
done 👍

Reply:
Yes, presets can be extended or overwritten so, still are flexible enough for users.